### ISSUE TITLE
[BUG]: Error in playwright.service.config with version 1.0.0-beta.5

### ISSUE THREAD

**Issue by marcusNumminen** [microsoft/playwright-testing-service#150]
**Describe the bug**
I get an error with the imports in the playwright.service.config.ts file when using the 1.0.0-beta.5 version

The error messages states:
_Module '"@azure/microsoft-playwright-testing"' has no exported member 'getServiceConfig'. Did you mean to use 'import getServiceConfig from "@azure/microsoft-playwright-testing"' instead?ts(2614)_

_Module '"@azure/microsoft-playwright-testing"' has no exported member 'ServiceOS'. Did you mean to use 'import ServiceOS from "@azure/microsoft-playwright-testing"' instead?ts(2614)_

The suggestions don't work as well....

If I use version  1.0.0-beta.4 then it works fine

**To Reproduce**
1. in package.json use 1.0.0-beta.5 for the module "@azure/microsoft-playwright-testing",
2. run npm install
3. Check the playwright.service.config.ts file

**Expected behavior**
no issues should be found in the file (using the default generated version)

**Screenshots**
![Image](https://github.com/user-attachments/assets/2f67450a-cf70-4c46-94cd-278ccf08769a)

**Setup information (please complete the following information):**
 - Azure Subscription Id:
 - Workspace Name:
 - Service trace Id (if available):
 - Log snippet (if available):
 - Browser on which tests are executed (if applicable):
 - Time range (in PST) when the issue was experienced:
 - Playwright Version: 1.48.2
- Client machine OS: Windows
- OS requested on Service:

**Additional context**
Node version 18.18.0
npm version: 9.8.1

**Comment by perezdev-ish** [comment]
I'm experiencing the same issue.

**Comment by Sid200026** [comment]
Hi @marcusNumminen, @perezdev-ish 

I'm unable to repro the scenario when running the [sample test suite](https://github.com/microsoft/playwright-testing-service/tree/main/samples/get-started) on Windows with node 18.18.0  using @azure/microsoft-playwright-testing@1.0.0-beta.5. Both ```ServiceOS``` and ```getServiceConfig``` are exported under @azure/microsoft-playwright-testing - [API Reference](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/playwrighttesting/microsoft-playwright-testing/review/microsoft-playwright-testing.api.md)

**Could you try out the below suggestions**

- Delete package-lock.json and node_modules and do a fresh install via ```npm install```
- If the issue persists, can you try cloning the [sample test suite](https://github.com/microsoft/playwright-testing-service/tree/main/samples/get-started), change the @azure/microsoft-playwright-testing version to 1.0.0-beta.5 and try running the tests.

**If the above suggestions doesn't work, could you share the below details**

- Does your test suite contain any other configuration files like ```tsconfig.json``` which might affect type resolution
- Output of ```npm ls @azure/microsoft-playwright-testing```
- Output of ```tsc --version```
- The content of ```package.json```

**Comment by perezdev-ish** [comment]
@Sid200026 the sample test suite kinda works. maybe there's a conflict with one of my other packages. I'm checking. but I also updated the sample packages from these:

"@azure/microsoft-playwright-testing": "^1.0.0-beta.3" -- "@azure/microsoft-playwright-testing": "^1.0.0-beta.5"
"@playwright/test": "^1.43.0" --> "@playwright/test": "^1.49.0"

And now I get this error in the config:

```
Argument of type 'PlaywrightTestConfig<{}, {}>' is not assignable to parameter of type 'PlaywrightConfigInput'.
  Types of property 'globalSetup' are incompatible.
    Type 'string | string[] | undefined' is not assignable to type 'string | undefined'.
      Type 'string[]' is not assignable to type 'string'.ts(2345)
```

**Comment by Sid200026** [comment]
```
And now I get this error in the config:
```
Please refer to https://github.com/microsoft/playwright-testing-service/issues/151. We will have a fix published in a couple of days. In the meantime, to mitigate it, consider downgrading playwright version to 1.48 or below.

```
maybe there's a conflict with one of my other packages
```
If these are public npm packages, could you share the list of packages?

**Comment by Sid200026** [comment]
Hi @diginikkari, thanks for pointing out the difference in the types file generated in 1.0.0-beta.5. We have reached out to the respective team to get it resolved. Will update this thread, once we have a fix along with ETA.

**Comment by Sid200026** [comment]
We have identified the root cause - https://github.com/Azure/azure-sdk-for-js/pull/31964. The fix will be released by 28th Nov, Thursday.

**Comment by Sid200026** [comment]
We have published an alpha version with the fix - 1.0.0-alpha.20241127.1. We will publish the beta version on 28th Nov.

**Comment by Sid200026** [comment]
We have released 1.0.0-beta.6 with the fix, please reopen this thread if the issue persists.
